config/routes.rb

```diff
@@ -1,372 +0,0 @@
-# frozen_string_literal: true
```

Comment:
If you removed all of these files, the entire app would not work, was there a reason they were removed?